site/docs/components/progress/accessibility.mdx

```diff
@@ -0,0 +1,16 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Accessibility
+data:
+  $ref: ./#/data
+---
+
+Improve accessibility by customizing the `ariaLabel` prop to provide additional context about what is loading. For example:
+
+```jsx
+<LinearProgress ariaLabel="loading settings panel 0%" />
```

Comment:
`ariaLabel` -> `aria-label`

Given "0%" is used, should `value` be used?

Reply:
0 is default